phpunit/block-supports/layout-test.php

```diff
@@ -101,4 +113,52 @@ function test_outer_container_not_restored_for_aligned_image_block_with_themejso
 
 		$this->assertSame( $expected, gutenberg_restore_image_outer_container( $block_content, $block ) );
 	}
+
+	/**
+	 * Tests that layout CSS is enqueued.
+	 */
+	public function test_should_not_enqueue_stored_styles() {
+		global $wp_styles;
+		gutenberg_get_layout_style(
+			'.tested',
+			array(
+				'type'        => 'default',
+				'contentSize' => '800px',
+				'wideSize'    => '1000px',
+			),
+			true,
+			null,
+			false,
+			'0.5em',
+			null,
+			false
+		);
+		gutenberg_enqueue_stored_styles();
+
+		$this->assertNull( $wp_styles );
+	}
+
+	/**
+	 * Tests that layout CSS is enqueued.
+	 */
+	public function test_enqueue_stored_layout_styles() {
+		global $wp_styles;
+		$layout_styles = gutenberg_get_layout_style(
+			'.test',
+			array(
+				'type'        => 'default',
+				'contentSize' => '800px',
+				'wideSize'    => '1000px',
+			),
+			true,
+			null,
+			false,
+			'0.5em',
+			null,
+			true
+		);
+		gutenberg_enqueue_stored_styles();
+
+		$this->assertEquals( array( $layout_styles ), $wp_styles->get_data( 'core-block-supports', 'after' ) );
```

Comment:
Tests are passing locally, but I think I need to deregister this style for the suite to pass. They're also being testing in phpunit/script-loader.php

I tried

```php
wp_styles()->remove( 'core-block-supports' );
wp_deregister_style( 'core-block-supports' );
```

None seem to work... 🤔